pyiron/thermodynamics/hessian.py

```diff
@@ -18,40 +18,117 @@
 
 class HessianJob(GenericInteractive):
     def __init__(self, project, job_name):
-        super(GenericInteractive, self).__init__(project, job_name)
+        super(HessianJob, self).__init__(project, job_name)
         self.__version__ = "0.0.1"
         self.__name__ = "HessianJob"
-        self.interactive_cache = {
```

Comment:
Why is this removed?

Reply:
Because since the introduction of `interactive_output_functions`, `interactive_cache` is not necessary anymore.